pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -112,6 +112,13 @@ let
   gccForLibs_solib = getLib gccForLibs
     + optionalString (targetPlatform != hostPlatform) "/${targetPlatform.config}";
 
+  libcxx =
```

Comment:
Can't you just move this into the default expression of `libcxx` in the args set? Or does something *explicitly* pass `libcxx = null`?